CommonTools/Utils/interface/DynArray.h

```diff
@@ -0,0 +1,31 @@
+#ifndef CommonTools_Utils_DynArray_H
+#define CommonTools_Utils_DynArray_H
+
+template<typename T>
+class DynArray {
+   T * a=nullptr;
```

Comment:
From coding rules: "The public, protected and private sections of a class must be declared in that order"